src/components/action-list/action-list.html

```diff
@@ -0,0 +1,160 @@
+            <div class="sg-icon-as-button__hole">
+              <div class="sg-overlayed-box">
+                <svg class="sg-icon sg-icon--x24 sg-icon--adaptive">
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-messages"></use>
+                </svg>
+              </div>
+            </div>
+          </button>
+        </div>
+        <div class="sg-actions-list__hole sg-actions-list__hole--no-spacing sg-actions-list__hole--container">
+          <button class="sg-icon-as-button sg-icon-as-button--light sg-icon-as-button--transparent">
+            <div class="sg-icon-as-button__hole">
+              <div class="sg-overlayed-box">
+                <svg class="sg-icon sg-icon--x24 sg-icon--adaptive">
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-notifications"></use>
+                </svg>
+                <div class="sg-overlayed-box__overlay">
+                  <div class="sg-badge sg-badge--peach sg-badge--rounded">
+                    <div class="sg-text sg-text--emphasised sg-text--light sg-text--xsmall">
+                      1
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </button>
+        </div>
+        <div class="sg-actions-list__hole sg-actions-list__hole--no-spacing sg-actions-list__hole--container">
+          <button class="sg-icon-as-button sg-icon-as-button--light sg-icon-as-button--transparent">
+            <div class="sg-icon-as-button__hole">
+              <div class="sg-overlayed-box">
+                <svg class="sg-icon sg-icon--x24 sg-icon--adaptive">
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-friends"></use>
+                </svg>
+              </div>
+            </div>
+          </button>
+        </div>
+        <div class="sg-actions-list__hole sg-actions-list__hole--no-spacing sg-actions-list__hole--container">
+          <div class="sg-icon-as-button sg-icon-as-button--transparent js-panel-button-icon">
+            <div class="sg-icon-as-button__hole">
+              <div class="sg-avatar sg-avatar--small  ">
+                <div class="sg-avatar__image sg-avatar__image--icon">
+                  <svg class="sg-icon sg-icon--x24 sg-icon--gray">
+                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-profile"></use>
+                  </svg>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action centered full width elements</h3>
+  </aside>
+
+  <div class="docs-block__content">
+    <div class="docs-block__contrast-box">
+      <div class="sg-actions-list">
+        <div class="sg-actions-list__hole--grow sg-actions-list__hole--full-width-centered">
```

Comment:
since there is `grow`, why do we need `full-width`? None of these classes say that this type forces column layout and row is the default.